pipeline/01-train.R

```diff
@@ -70,7 +70,7 @@ message("Initializing LightGBM model")
 # model arguments, which are provided by parsnip's boost_tree()
 lgbm_model <- parsnip::boost_tree(
   stop_iter = params$model$parameter$stop_iter,
-  trees = params$model$parameter$num_iterations
+  trees = tune()
```

Comment:
I switched `num_iterations` and `learning_rate` to tunable parameters for now. We'll see how the CV performs with them unlocked.